src/hooks/use-toast.ts

```
@@ -113,82 +116,84 @@ export const reducer = (state: State, action: Action): State => {
       }
       return {
         ...state,
         toasts: state.toasts.filter((t) => t.id !== action.toastId),
-      }
+      };
   }
-}
+};
 
-const listeners: Array<(state: State) => void> = []
+const listeners: Array<(state: State) => void> = [];
```

ESLint

[error] Array type using 'Array<T>' is forbidden. Use 'T[]' instead.

(@typescript-eslint/array-type)